nevergrad/optimization/test_optimizerlib.py

```diff
@@ -164,6 +164,9 @@ def test_optimizers_suggest(name: str) -> None:  # pylint: disable=redefined-out
         candidate = optimizer.ask()
         try:
             optimizer.tell(candidate, 12)
+            # The optimizer should recommend its suggestion, except for a few optimization methods:
```

Comment:
`suggest` is something completely different, so the comment is misleading, what do you want to try here?

Reply:
it's suggest indeed, didnt get it sorry